pkgs/development/python-modules/lancedb/default.nix

```diff
@@ -0,0 +1,61 @@
+{ buildPythonPackage
+, aiohttp
+, attrs
+, cachetools
+, click
+, deprecation
+, fetchPypi
+, lib
+, overrides
+, pydantic
+, pylance
+, pythonRelaxDepsHook
+, pyyaml
+, requests
+, retry
+, semver
+, setuptools
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "lancedb";
+  version = "0.3.3";
+
+  format = "pyproject";
```

Comment:
Should be `pyproject = true;` now (see https://github.com/NixOS/nixpkgs/commit/39c64e232328e09284c766d2b56f2d680e511d41).